.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+---
+repos:
+  - repo: https://github.com/adrienverge/yamllint
```

Comment:
Let's separate repos items for better readability?

Reply:
Done